### ISSUE TITLE
parallelize multi-arch container image builds

### ISSUE THREAD

**Issue by verokarhu** [dalinjun/build-tools#38]
In order to **better take use of cluster resources**
As a **developer**
I want to **run multi-arch builds in parallel**

***

- [ ] Upgrade to Buildah 1.23
- [ ] Add flags for running multiple jobs and multiple platforms at one time

***

See https://github.com/containers/buildah/blob/main/docs/buildah-build.md

**Comment by verokarhu** [comment]
This has been superseded by https://github.com/dalinjun/tektonator
